### ISSUE TITLE
Incorrect resume

### ISSUE THREAD

**Issue by yurkovak** [Megvii-BaseDetection/YOLOX#1318]
Hi! Thanks for a great repo! 
I noticed that resume is not done right when training with EMA. In this case only EMA model's weights are saved, so on resume both the model and the EMA model are initialized from the EMA weights, which is not the same as not pausing training in the first place. I believe both model's and EMA model's weights should be saved and re-loaded separately.

**Comment by FateScript** [comment]
@yurkovak, yes, you are correct . I will mark this issue as `bug fix`. Thanks a lot.
